src/components/overview/ProductInfo/ProductInfo.js

```diff
@@ -5,21 +5,23 @@ import StyleSelector from '../StyleSelector/StyleSelector.js'
 const ProductInfo = ({products}) => {
 
   const [price, setPrice] = useState('');
+  const [skus, setSkus] = useState({});
   useEffect(() => {
     setPrice(products.default_price);
+   // setSkus(products.results[0].skus)
```

Comment:
In general, don't leave commented out lines like this in the code. Just a small thing, but it'll lead to confusion for other developers in the future as to why it's still there/what it's supposed to do/why they may need to uncomment it/etc.

If for whatever reason you *do* need to leave commented out code in your file, put a comment above it explaining why it's still there.